mathlibtools/lib.py

```diff
@@ -639,6 +639,33 @@ def import_graph(self) -> 'ImportGraph':
         self._import_graph = G
         return G
 
+    def reduce_imports(self, file: str) -> Iterable[List[Any]]:
```

Comment:
I don't understand how the type can be mysterious here. Why `Any`?

Reply:
You are right (of course!), it was simply my inexperience with typing made me think that as networkx doesn't have type annotations this was the best possible.